source/cItem.cpp

```diff
@@ -542,6 +543,27 @@ auto CItem::SetSpawnerList( bool newValue ) -> void
 	UpdateRegion();
 }
 
+//o------------------------------------------------------------------------------------------------o
+//|	Function	-	CBaseObject::GetArtifactRarity()
+//|					CBaseObject::SetArtifactRarity()
+//|	Date		-	9 May, 2024
+//o------------------------------------------------------------------------------------------------o
+//|	Purpose		-	Gets/Sets the Artifacts Rarity of the object
+//o------------------------------------------------------------------------------------------------o
+SI16 CItem::GetArtifactRarity( void ) const
+{
+	return artifactRarity;
+}
+void CItem::SetArtifactRarity( SI16 newValue )
+{
+	artifactRarity = newValue;
+
+	if( CanBeObjType( OT_ITEM ))
```

Comment:
As with the other PR, because we're already in CItem, this should be reducable to just `UpdateRegion();`

We're in Item, so can always be item, and shouldn't need to static cast.